docs/wiki/Configuration:-Debug-Options.md

```diff
@@ -18,6 +18,8 @@ debug {
     disable-direct-scanout
     restrict-primary-scanout-to-matching-format
     render-drm-device "/dev/dri/renderD129"
+    disable-drm-device "/dev/dri/renderD128"
+    disable-drm-device "/dev/dri/renderD130"
```

Comment:
Maybe it should say "ignore-drm-device"? Idk, what is it called in other compositors?